tests/testlib/wakunode.nim

```diff
@@ -13,8 +13,8 @@ import
   ../../../waku/node/peer_manager,
   ../../../waku/waku_enr,
   ../../../waku/waku_discv5,
+  ../../apps/wakunode2/external_config,
   ../../apps/wakunode2/internal_config,
-  ../wakunode2/test_app,
```

Comment:
When importing `defaultTestWakuNodeConf` from `test_app` into other test suites, the `test_app` suite was being re-run as part of the suites importing it. For easier `defaultTestWakuNodeConf`'s reusability, moved the function to `external_config`, mimicking  `newTestWakuNode`, which is reused by many test suites and is defined in `wakunode.nim`

Reply:
Ah, made a comment about this elsewhere before seeing this. `wakunode.nim` is a `testlib` - in other words, it's designed to be a utility module for functions useful in tests. `newTestWakuNode` `defaultTestWakuNodeConf` etc. shouldn't be defined in any "production" modules (test code should remain completely separate. The way to prevent `test_app` from being run multiple times for each import is to move `defaultTestWakuNodeConf` to a test helper module. I'd suggest just adding it to `testlib/wakunode.nim` and importing in both `test_app` and `test_waku_netconfig`